style.css

```diff
@@ -0,0 +1,775 @@
+.navbar-bg {
+    background: #0d4f9a;
+}
+
+body {
+    top: 0;
+    background: #ffffff;
+    font-family: var(--font-stack-body, "Lato", "HelveticaNeue", "Helvetica Neue", sans-serif);
```

Comment:
format style as top on related to tags like body or p then follow them by className